internal/cloud/backend_taskStages.go

```diff
@@ -97,8 +97,15 @@ func (b *Cloud) runTaskStage(ctx *IntegrationContext, output IntegrationOutputWr
 		case tfe.TaskStagePending:
 			// Waiting for it to start
 			return true, nil
+		case tfe.TaskStageRunning:
+			_, e := processSummarizers(ctx, output, stage, summarizers, errs)
+			if e != nil {
+				errs = e
+			}
```

Comment:
minor. won't stop approval. Could probably clean this up a bui

```suggestion
			if _, e := processSummarizers(ctx, output, stage, summarizers, errs); e != nil {
				errs = e
			}
```